Sources/WalletConnectRelay/Dispatching.swift

```diff
@@ -82,9 +82,9 @@ final class Dispatcher: NSObject, Dispatching {
 
     private func dequeuePendingTextFrames() {
         while let frame = textFramesQueue.dequeue() {
-            send(frame) { error in
+            send(frame) { [weak self] error in
```

Comment:
would `unowned self` be better?

Reply:
replaced weak with unowned everywhere in Dispatcher. Should work as we have Relay singleton